service-front/web/main.scss

```
@@ -12,3 +12,33 @@ $govuk-assets-path: "../assets/";
 .moj-banner--date {
   max-width: 150px;
 }
+
+.govuk-inset-text-blue {
```

We shouldn't define classes with a `govuk-` prefix in case they're ever implemented by the Design System. Instead, we tend to use the prefix `app-`
```suggestion
.app-inset-text-blue {
```